flowback/poll/services/vote.py

```diff
@@ -335,3 +361,35 @@ def save_participants_count(voting_type_class):
         poll.status = 1 if poll.participants > total_group_users * quorum else -1
         poll.result = True
         poll.save()
+
+
+    if poll.dynamic:
+        quorum_completed = poll.participants > total_group_users * quorum
+
+        if quorum_completed:
+            if poll.approval_minimum:
+                approval_minimum = poll.approval_minimum / 100
+                # Check if the approval_minimum is met
+                approval_minimum_completed = False
+                for proposal in poll.pollproposal_set.all():
+                    proposal_positive_votes = proposal.proposal_votes_info.get('positive_votes', 0)
+                    if proposal_positive_votes > poll.participants * approval_minimum:
```

Comment:
poll.participants * approval_minimum isn't correct, you should multiply approval_minimum with the total amount of voters

Reply:
@Gofven 
**poll. participants** are the total amount of votes, they are calculated before this point using this query:

```
   poll.participants = (
                mandate + PollVoting.objects.filter(poll=poll).all().count()
            )
```